sdk/core/core/inc/az_context.h

```diff
@@ -20,69 +20,112 @@
 typedef struct az_context az_context;
 
 /**
- * @brief An az_context value in a node in a tree that represents expiration times and key/value
+ * @brief An az_context value is a node in a tree that represents expiration times and key/value
```

Comment:
nit: Can we separate out the `az_context.h` related changes/docs into a separate PR, leaving this span focused? If it is not convenient/easy to do so, ignore it.

Reply:
Context is small compared to span.  Would prefer to do it at the same time

Reply:
I pulled it out.  I'll put it in a seperate PR